File: app/src/main/java/com/github/kotvertolet/youtubeaudioplayer/data/PlaylistWithSongs.java
package com.github.kotvertolet.youtubeaudioplayer.data;

import com.github.kotvertolet.youtubeaudioplayer.db.dto.PlaylistDto;
import com.github.kotvertolet.youtubeaudioplayer.db.dto.YoutubeSongDto;

import java.util.List;

public class PlaylistWithSongs {

    private PlaylistDto playlist;
    private List<YoutubeSongDto> songs;

    public PlaylistWithSongs() {
    }

    public PlaylistWithSongs(PlaylistDto playlist, List<YoutubeSongDto> songs) {
        this.playlist = playlist;
        this.songs = songs;
    }

    public PlaylistDto getPlaylist() {
        return playlist;
    }

    public void setPlaylist(PlaylistDto playlist) {
        this.playlist = playlist;
    }

    public List<YoutubeSongDto> getSongs() {
        return songs;
    }

    public void setSongs(List<YoutubeSongDto> songs) {
        this.songs = songs;
    }
}

File: app/src/main/java/com/github/kotvertolet/youtubeaudioplayer/data/PlaylistWithSongs.kt
package com.github.kotvertolet.youtubeaudioplayer.data

import com.github.kotvertolet.youtubeaudioplayer.db.dto.PlaylistDto
import com.github.kotvertolet.youtubeaudioplayer.db.dto.YoutubeSongDto

class PlaylistWithSongs {
    var playlist: PlaylistDto? = null
    var songs: List<YoutubeSongDto>? = null

    constructor()
    constructor(playlist: PlaylistDto?, songs: List<YoutubeSongDto>?) {
        this.playlist = playlist
        this.songs = songs
    }

}